idseq_dag/util/m8.py

```diff
@@ -599,7 +582,51 @@ def should_keep(hits):
             }
         }
 
-    with log.log_context("generate_taxon_count_json_from_m8", {"substep": "json_dump", "output_json_file": output_json_file}):
+    with log.log_context(
+        "generate_taxon_count_json_from_m8",
+        {"substep": "json_dump", "output_json_file": output_json_file}
+    ):
         with open(output_json_file, 'w') as outf:
             json.dump(output_dict, outf)
             outf.flush()
+
+
+def build_should_keep_filter(
+    deuterostome_path,
+    taxon_whitelist_path,
+    taxon_blacklist_path
+):
+
+    # See also HOMO_SAPIENS_TAX_IDS in idseq-web
+    taxids_to_remove = set(['9605', '9606'])
```

Comment:
Do we want to show other species within the genus Homo? Or remove all with genus == Homo?

Reply:
These were the values I put here are from HOMO_SAPIENS_TAX_IDS filtering in idseq-web, so if want to change our policy on Homo we should do it everywhere.

Reply:
Ok, thank you for clarifying!